docs/content/guides/operations/providers/aws-provider/howto-aws-provider-irsa/index.md

```diff
@@ -0,0 +1,130 @@
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Setup a supported Kubernetes cluster]({{< ref "/guides/operations/kubernetes/overview#supported-clusters" >}})
+  - You will need the cluster's OIDC Issuer URL. [EKS Example](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html)
+- [Create an IAM Policy] (https://docs.aws.amazon.com/eks/latest/userguide/associate-service-account-role.html)
+
+
+## Setup the AWS IAM Roles for Service Accounts(IRSA) for Radius
+
+To authorize Radius to connect to AWS using AWS IAM Roles for Service Accounts(IRSA), you should assign IAM roles to Kubernetes service accounts. 
+
+To associate an IAM role with a Kubernetes service account Create an IAM role and associate it with a Kubernetes service account.
+- Go to Identity and Access Management (IAM) on AWS portal and create a new role.
+![alt text](image.png)
+- Select `Trusted entity type` as `Web Identity` and `Identity Provider` as the cluster OIDC url.
+![alt text](image-1.png)
+- Select the policy to attach to your new role.
+- Add `Role Name` and create role using the default trust policy.
+- Update the Trust Policy to match to the below format.
+    ```
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Principal": {
+                    "Federated": "arn:aws:iam::<account-id>:oidc-provider/<oidc-url>"
+                },
+                "Action": "sts:AssumeRoleWithWebIdentity",
+                "Condition": {
+                    "StringEquals": {
+                        "<oidc-url>:aud": "sts.amazonaws.com",
+                        "<oidc-url>:sub": "system:serviceaccount:radius-system:ucp"
+                    }
+                }
+            },
+            {
+                "Sid": "Statement1",
+                "Effect": "Allow",
+                "Principal": {
+                    "Federated": "arn:aws:iam::<account-id>:oidc-provider/<oidc-url>"
+                },
+                "Action": "sts:AssumeRoleWithWebIdentity",
+                "Condition": {
+                    "StringEquals": {
+                        "<oidc-url>:aud": "sts.amazonaws.com",
+                        "<oidc-url>:sub": "system:serviceaccount:radius-system:applications-rp"
+                    }
+                }
+            }
+        ]
+    }
+    ```
+
+## Interactive configuration
+
+1. Initialize a new environment with [`rad init --full`]({{< ref rad_init >}}):
+
+   ```bash
+   rad init --full
+   ```
+
+1. Follow the prompts, specifying:
+   - **Namespace** - The Kubernetes namespace where your application containers and networking resources will be deployed (different than the Radius control-plane namespace, `radius-system`)
+   - **Add an AWS provider** - Enter your IAM Role ARN and pick a region to deploy your AWS resources to.
```

Comment:
Can you expand on the steps here to select AWS IRSA option in the interactive experience and provide the required details. Also how does a user get the IAM role arn?

Reply:
updated with details